packages/react/src/components/FEaaS/FEaaSWrapper.tsx

```diff
@@ -35,10 +35,10 @@ export const FEaaSComponent = (props: FEaaSComponentProps): JSX.Element => {
     <FEAAS.Component
       data={data}
       template={props.template}
-      cdn={props.params?.ComponentHostName}
-      library={props.params?.LibraryId}
-      version={props.params?.ComponentVersion}
-      component={props.params?.ComponentId}
+      cdn={props.params?.ComponentHostName as string}
+      library={props.params?.LibraryId as string}
+      version={props.params?.ComponentVersion as string}
+      component={props.params?.ComponentId as string}
```

Comment:
Type cast is not really needed here, in case of `library` and `component` we can just fallback to empty string. Also update the existing [unit test](https://github.com/Sitecore/content-sdk/blob/45c3ea7cdadb778e1bf283801dc56895670158d5/packages/react/src/components/FEaaS/FEaaSWrapper.test.tsx#L60)
```suggestion
      cdn={props.params?.ComponentHostName}
      library={props.params?.LibraryId ?? ''}
      version={props.params?.ComponentVersion}
      component={props.params?.ComponentId ?? ''}
```